client/python/samples/utils/utils.py

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from ovmsclient import make_grpc_metadata_request
+import cv2
+import numpy as np
+
+
+def read_paths(images_dir):
+    with open(images_dir) as f:
+        return f.readlines()
+
+
+def load_image(path):
+    with open(path, 'rb') as f:
+        return f.read()
+
+
+# get single input and output name from model metadata
+def get_single_metadata(client, model_name, model_version):
+    metadata_request = make_grpc_metadata_request(model_name, model_version)
+    metadata = client.get_model_metadata(metadata_request)
+    metadata_dict = metadata.to_dict()
+    version = next(iter(metadata_dict))
+    input_name = next(iter(metadata_dict[version]['inputs']))  # by default resnet has one input and one output
+    output_name = next(iter(metadata_dict[version]['outputs']))
+    return input_name, output_name
+
+
+# read .txt file with listed images into ndarray in model shape
+def read_imgs_as_ndarray(images_dir, height, width):
```

Comment:
Can we just read all jpg and png files in the `images_dir` path instead of relying on .txt file?